modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/AbstractIndexStorageTest.java

```diff
@@ -79,9 +85,11 @@ public abstract class AbstractIndexStorageTest<S extends IndexStorage, D extends
 
     protected static final int TEST_PARTITION = 12;
 
-    protected static final String INDEX_NAME = "TEST_IDX";
+    protected static final String TABLE_NAME = "FOO";
+
+    protected static final String PK_INDEX_NAME = pkIndexName(TABLE_NAME);
 
-    protected static final String TABLE_NAME = "foo";
+    protected static final String INDEX_NAME = "TEST_IDX";
```

Comment:
Did you change the order of constants manually, or it's been done automatically by IDEA somehow?

Reply:
I don't see anything wrong with this and it's not prohibited.